Rock.Slingshot/SlingshotImport/SlingshotImporter.cs

```diff
@@ -1334,7 +1419,7 @@ private List<Rock.Slingshot.Model.PersonImport> GetPersonImportList()
 
                 if ( !string.IsNullOrEmpty( slingshotPerson.Salutation ) )
                 {
-                    personImport.TitleValueId = this.PersonTitleValues[slingshotPerson.Salutation]?.Id;
+                    personImport.TitleValueId = this.PersonTitleValues[System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase( slingshotPerson.Salutation.ToLower() )]?.Id;
```

Comment:
@JoshuaMaiorino Depending on why this done, I discovered there is a way to make a 'Case-Insensitive' dictionary that would avoid the ToUpper,ToLower trick that we often use for dictionary lookup.  Maybe you did this for a different reason since you are also doing a ToTitleCase.  Otherwise, the case-insensitive dictionary is pretty handy.  https://stackoverflow.com/a/13988953/1755417